react-native-aztec/android/src/main/java/org/wordpress/mobile/ReactNativeAztec/ReactAztecManager.java

```diff
@@ -202,6 +203,11 @@ public Map getExportedCustomDirectEventTypeConstants() {
 
     @ReactProp(name = "text")
     public void setText(ReactAztecText view, ReadableMap inputMap) {
+        if (inputMap.hasKey(LINK_TEXT_COLOR_KEY)) {
+            int color = Color.parseColor(inputMap.getString(LINK_TEXT_COLOR_KEY));
+            setLinkTextColor(view, color);
```

Comment:
This means that we will be constructing new `LinkFormatter` and `LinkStyle` objects on text change, right? Seems like that is not ideal from a performance perspective, although any impact would probably be negligible. Still, maybe it's worth only updating the link formatter when the link color has changed. I do not feel strongly about this.

Reply:
> Still, maybe it's worth only updating the link formatter when the link color has changed. 

So are you thinking something along the lines of this? 
```suggestion
        int color = Color.parseColor(inputMap.getString(LINK_TEXT_COLOR_KEY));
        if (view.linkFormatter.color != color) {
            setLinkTextColor(view, color);
```

*Note* I didn't test this suggestion out yet.

Reply:
Yes, that's pretty much what I was imagining.

Reply:
I'm good with that 👍 I'll work on making that change.